test/e2e.yaml

```diff
@@ -40,6 +40,8 @@ verify:
     interval: 10000
   cases:
     - expected: expected/page-path.yml
-      query: swctl --display yaml --base-url=http://127.0.0.1:${ui_8080}/graphql endpoint ls --service-id dGVzdC11aQ==.1
+      query: swctl --display yaml --base-url=http://127.0.0.1:${ui_8080}/graphql endpoint ls --service-name dGVzdC11aQ==.1
```

Comment:
Why use `-name`, but actually pass the id?

Reply:
The current `swctl` query does not contain browser service

Reply:
Yes, I know. We need some patches on the CLI side.